spec/controllers/v0/sign_in_controller_spec.rb

```diff
@@ -1177,7 +1260,14 @@
       create(:validated_credential, user_verification: user_verification, client_id: client_id)
     end
     let(:client_id) { SignIn::Constants::ClientConfig::CLIENT_IDS.first }
-    let(:error_context) { { refresh_token: refresh_token, anti_csrf_token: anti_csrf_token } }
+    let(:error_context) do
+      { refresh_token: refresh_token, anti_csrf_token: anti_csrf_token,
+        type: type, client_id: client_id_value, loa: loa }
+    end
+
+    before do
```

Comment:
Can remove this after you `let!(:user)` above

Reply:
I tried removing this and the other `before` block after adding the bang! to the User class creation but it still was not getting instantialized, causing the specs to fail. Any ideas why?

Reply:
Ah, I see, you also need to match up the uuids, try this:
`let!(:user) { create(:user, uuid: user_verification.user_account.id) }`
(At least for the token ones, that works)